tests/test_load_data.py

```diff
@@ -307,6 +310,31 @@ def test_grid_netcdf(self):
 
         self.assertIsNone(run_smrf(config))
 
+class TestLoadTopo(unittest.TestCase):
+    @classmethod
+    def setUp(self):
+        base = dirname(smrf.__file__)
+        self.test_dir = abspath(join(base, '../', 'tests'))
+        topo_config = {
+            'filename': join(self.test_dir, 'RME/topo/topo.nc'),
+            'northern_hemisphere':True,
+        }
+
+        self.topo = loadTopo.topo(
+            topo_config,
+            calcInput=False,
+            tempDir= join(self.test_dir, 'RME/output')
+        )
+
+    def test_auto_calc_lat_lon(self):
+        '''
+        Test we calculate the basin lat long correctly
+        '''
+        # Original RME
+        # basin_lon:                     -116.7547
+        # basin_lat:                     43.067
+        self.assertTrue(self.topo.basin_lat == 43.06475372378507)
```

Comment:
What I'm finding with floating point comparisons is that it doesn't always come out the same with different operating systems. There is [assertAlmostEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual) in the `unittest` but also in numpy there is a similar version.

Reply:
I had that in there but I remembered that we were going for more exact so I went back. I am fine with it, how many decimal places do you want?

Reply:
When I run other tests that compares decimals on my Mac, I was able to match the first three. Speaking for the OS X users out there :-)

Reply:
I was thinking of you Joe! Lets try with the standard comparison with 7 decimals until it becomes too restrictive. Joe's was for the gold files which have a lot of calculations on them.

Reply:
I agree and start bigger here. Lat/Lon calculations hopefully have more agreement on decimals (Mac's would otherwise navigate to different places with low decimal matching)